### ISSUE TITLE
Support for Synology DiskStation DS220+ (v112.1.0)

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2172]
As an assumed environment, the Docker environment will be improved on the premise of DS220+.

> DS220+ : Very popular 2-bay NAS in the Japanese market. Ranked No 2 on ranking sites. (Inexpensive NAS with poor application server functions is often ranked first). In terms of specs, DS220+ is enough for Jpsonic.

 - Even now, it is relatively easy to reach out to run Jpsonic on Docker.
 - ~~However, the Docker image needs some additional improvements such as configuring permissions.~~ It is desirable that a new production.yml for DS220+ be maintained.
   - Because it uses DLNA, it is suitable to run in host mode.
   - Also, Docker's GUI is poor and requires a few command operations. Need an explanation on the wiki?

___

Jpsonic was able to scan at a sufficient speed when the assumed environment was DS220+. The speed is sufficiently practical. (Improving Docker set-up will make it more user-friendly)

Therefore, it seems better to change the milestone as follows :

v112.1.0 (as little as possible, as soon as possible!)
↓
v112.2.0 (as little as possible, as soon as possible!)
↓
v113.0.0 (It's pretty hard, but let's do our best to improve the podcast!)

 - [jpsonic 112.2.0](https://github.com/tesshucom/jpsonic/milestone/65) plans to parallelize scans
     - As expected from the beginning, video scanning seems to be better with a parallelization option. 
     - Lower the load of jpsonic 112.2.0 as much as possible. Unimportant speed improvements can be deferred. Because, Podcast improvements included in [jpsonic 113.0.0](https://github.com/tesshucom/jpsonic/milestone/63) are more urgent!
- v112.2.0 was pre-planned to handle parallelization. Before that, establish a new milestone [112.1.0](https://github.com/tesshucom/jpsonic/milestone/67)
  - Clarify installation instructions, including providing a new production.yml for DS220+.
  - To speed things up, you need a measurement environment that is both fair and specific. The DS220+ relatively satisfies that requirement. In other words, the preparation of the DS220+ operating environment must be done before speed-related improvements.

**Comment by tesshucom** [comment]
In #2175, DS220+ specific introduction procedure was shown. When v112.1.0 is released, it would be nice to add a snapshot and create a documentation page on the wiki.
